### PR TITLE
fix: verification issue [APE-964]

### PR DESCRIPTION
**Pull request by antazoey**
### What I did

* regression from rate limiting
* clean api key from exceptions
* timeout issue
* issue where falsely didnt count already verified contracts as ok

### Checklist

- [ ] Passes all linting checks (pre-commit and CI jobs)
- [ ] New test cases have been added and are passing
- [ ] Documentation has been updated
- [ ] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

File: ape_etherscan/client.py [hunk 1]
```diff
@@ -183,16 +183,24 @@ def _request(
         headers = headers or self.DEFAULT_HEADERS
         for i in range(self._retries):
             response = self.session.request(
-                method.upper(), self.base_uri, headers=headers, params=params, data=data
+                method.upper(),
+                self.base_uri,
+                headers=headers,
+                params=params,
+                data=data,
+                timeout=1064,
```
Comment (fubuloubu) [inline ape_etherscan/client.py:191]: weird timeout choice

Comment (antazoey) [reply ape_etherscan/client.py:191]: give one that you find less weird plz

Comment (antazoey) [reply ape_etherscan/client.py]: i made less weird but gimme 1 if you still dont like it.

Comment (fubuloubu) [reply ape_etherscan/client.py]: That works, was more curious if there was any special meaning to 1064 that needed to be documented

**Review by fubuloubu — APPROVED** [review]

**Pull request merged**